app/main/posts/views/post-view-list.html

```diff
@@ -1,6 +1,8 @@
-<div class="main-col">
+<div class="main-col" id='top'>
```

Comment:
is this an id we need because of the PL?

Reply:
if it's not absolutely necessary that this is named 'top', I think a prefixed id like post-view-list-top would be better. The reason is that ids being unique, you want to minimize opportunities for collusion in IDs.

Reply:
Good point! Done.